CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Unreleased
 
 - On X11, fixed panic caused by dropping the window before running the event loop.
+- on macOS, added a new simple fullscreen mode which does not require a separate space
```

Comment:
It would be awesome if this mentioned the method name somewhere (`WindowExt::set_simple_fullscreen`)